examples/chat/chat.py

```diff
@@ -12,34 +13,28 @@
 
 async def read_data(stream):
     while True:
-        read_string = (await stream.read()).decode()
-
-        if not read_string:
-            return
-
-        if read_string != "\n":
-            # Green console colour: 	\x1b[32m
-            # Reset console colour: 	\x1b[0m
-            print("\x1b[32m%s\x1b[0m> " % read_string, end="")
+        read_string = await stream.read()
+        if read_string is not None:
+            read_string = read_string.decode()
+            if read_string != "\n":
+                # Green console colour: 	\x1b[32m
+                # Reset console colour: 	\x1b[0m
+                print("\x1b[32m %s\x1b[0m > " % read_string, end="")
```

Comment:
Can we change this line to `print("\x1b[32m%s\x1b[0m" % read_string, end="")`? The ` `  and `>` can sometimes be confusing when a user is sending two consecutive messages. Thanks

Reply:
`print("\x1b[32m %s\x1b[0m " % read_string, end="")`  
I have removed the `>`, but have kept the spaces. This is because, the color codes do not work on Windows, and the raw `\x1b[0m` and `x1b[32m` look odd, and get mixed up with the message. The space helps to separate them.

Reply:
I see, we will fix the UI issues later at some point. thanks!